OBAKit/UI/OBATheme.m

```diff
@@ -52,6 +54,43 @@ + (void)setAppearanceProxies {
     [[[UITableViewCell appearance] contentView] setPreservesSuperviewLayoutMargins:YES];
 }
 
++ (UIView *)createNavbarTitleViewWithTitle:(NSString *)title subtitle:(NSString *)subtitle andStyle:(OBAAppearanceNavBarTitleViewStyle)style {
```

Comment:
I think we're going to be better off with a custom control instead of composing a new view out of two labels. Maybe `OBANavigationTitleView`? It'd be a subclass of `UIView` and expose two `MarqueeLabel`s called `titleLabel`, and `subtitleLabel`. The two marquee label properties should be declared as `readonly` in the .h file like this:

```
@interface OBANavigationTitleView: UIView
@property(nonatomic,strong,readonly) MarqueeLabel *titleLabel;
@property(nonatomic,strong,readonly) MarqueeLabel *subtitleLabel;
```

And then they can be marked `readwrite` in the `.m` file. Let me know if you haven't seen this before and I'll point you in the direction of an example.

Also, please make sure to use the `UIFont`s defined in `OBATheme` so that font sizes change dynamically based upon the system.